Mathlib/Analysis/Asymptotics/TVS.lean

```diff
@@ -0,0 +1,189 @@
+/-
+Copyright (c) 2023 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Analysis.Asymptotics.Asymptotics
+import Mathlib.Analysis.Convex.EGauge
+import Mathlib.Analysis.LocallyConvex.BalancedCoreHull
+import Mathlib.Analysis.Seminorm
+import Mathlib.Tactic.Peel
+import Mathlib.Topology.Instances.ENNReal
+
+/-!
+# Asymptotics in a Topological Vector Space
+
+This file defines `Asymptotics.IsLittleOTVS` as a generalization of `Asymptotics.IsLittleO` from
+normed spaces to toplogical spaces.
+
+This frees the user from having to chose a canonical norm, at the expense of having to pick a
+specific base ring.
```

Comment:
@urkud, can you write something here comparing to https://en.wikipedia.org/wiki/Fr%C3%A9chet_derivative#Generalization_to_topological_vector_spaces ?

Reply:
I wrote something. Feel free to update.